charts/datahub/subcharts/datahub-frontend/templates/user-secrets.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.defaultUserCredentials }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ printf "%s-user-secret" .Release.Name }}
+  labels:
+    {{- include "datahub-frontend.labels" . | nindent 4 }}
+type: Opaque
+data:
+  user.props: {{ .Values.defaultUserCredentials | b64enc | quote }}
```

Comment:
It might be better to leave the `defaultUserCredentials` empty and generate a random password by default on initial install. This way we can avoid the issue of even having a set of default credentials at install. What do you think?

Reply:
Random password generation also great, but I think it should be possible for the deployer to set the password of the initial user.

Also, I think `defaultUserCredentials` prop should be **non-empty**. Because `"datahub"`, the datahub's initial password is so simple, and widely known, so if deployer fotgot the set or change the initial password, any attackers can access as initial Admin user.

Reply:
For a random password, it can be good to add a passwor random generation option 👍

Reply:
Hello, @david-leifker, I added `randomAdminPassword` option to generate random 32 length password! It's a default option, but if deployer want to set `user.props` file manually, he can use `manualValue` option instead.

Reply:
Thank you!